### ISSUE TITLE
Extras/Varia encoding and guidelines

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/guidelines#42]
from https://github.com/BetaMasaheft/Documentation/issues/1128

by @PietroLiuzzo :

> If I remember correctly, the distinction between extra and additions is an inheritance of Ethio-SPaRe which we maintained grouping the items in additions with different `xml:id`s whereas the only semantic distinction in the TEI is between `msItem`s and `additions//item` 
> 
> The section [Varia](https://betamasaheft.eu/Guidelines/?q=extra&id=additionsVaria)  of this page does need a lot of editing indeed, as there are a number of things in that list which we do differently

> I forgot to add, that because of the above and the distinction made in the guidelines currrently I would not see too much of a problem in using the currently allowed types for any item in additions also for extras. It is instead questionable how to decide on the level of "triviality" which the guidelines now suggest so that perhaps this point needs to be specified and better described as well.

The guidelines on encoding extras could be edited. It might make sense to use more @type with `<desc>` here, and possibly to group extras describing scribal practices. 

Does someone feel called to work on this? :)
@DenisNosnitsin1970 @eu-genia @DariaElagina @sologebre

**Comment by thea-m** [comment]
ideas for "extra" types:
- scribbles
- pen trials
- parchment repair (does this information rather belong to condition?)
- line take-ups/-downs
- tamallas-sign
- phenomena concerning corrections, omissions etc.
- substituted names
- indications for readings
- unfilled spaces left for rubrication
- cues for rubrication (maybe it would make sense to group information on rubrication in one place?)
- reduced script
- compressed script

**Comment by thea-m** [comment]
What do you think about this systematization? @DenisNosnitsin1970 ?
line take-ups/-downs, tamallas-sign, phenomena concerning corrections, omissions etc., reduced and compressed script, unfilled spaces left for rubrication, cues for rubrication could possibly be grouped into a larger scribal practices category.
If there are no important conceptual objections, I would suggest to prepare a guidelines page for this, which I will then submit for review.

**Comment by thea-m** [comment]
I still think that it would make sense to create categories for "extras" (either broader ones as "scribal practices" and or more narrow ones as "corrections", down to the exact type of correction, for example) and I still would be willing to draft a guidelines page, but only if there is a consesus that it will be worthwile.
What do you think?
@DenisNosnitsin1970 @DariaElagina 
@SusanneHummel @MassimoVilla @antobrita

**Comment by DenisNosnitsin1970** [comment]
Sorry, now I do not understand, what will be under "Varia"?

**Comment by thea-m** [comment]
The same as is now, the idea was to categorise it more formally (it often is categorised already in the head of the encoder, so I don't think that it would be a conceptually big step).
For example something like (hypothetical both from the content and the encoding):

```
<item xml:id="e1">
<desc type="ScribalPractices"/>

<item xml:id="e1_1">
<desc type="Corrections">Corrections are carried out throughout by erasures and marginal additions.</desc>
</item

<item xml:id="e1_2">
<desc type="RubricatorCues">Cues for the rubricator are written in the margins next to space left empty for rubrications.</desc>
</item

</item
```

**Comment by DenisNosnitsin1970** [comment]
This is possible and will be certainly  helpful, someone will need to fix up categories, and it will be an open list for a good while. For at least a half it will be scribal practicesm, but there will be a group "unclear".

**Comment by MassimoVilla** [comment]
I strongly agree with the need of a better formalization of the various phenomena currently encoded under "Varia", whose internal subgrouping (corrections, aids for readings, titles in the upper margin, minor scribbles, etc.) is already clear in our head but is left unspecified - at least formally - in the encoding practice. I also think that the list should be left open to new categories, so that new values might be added after discussion if needed.

**Comment by SusanneHummel** [comment]
I also absolutely agree. We just recently disussed exactly this point desired such suggestions. :)

**Comment by antobrita** [comment]
I do agree too. I would substitute in the list "indications for readings" with a more general "indications related to performance" (which is a broader category: it includes reading indications, but not exclusively).
